test/unit/core/mongodb_srv.test.js

```diff
@@ -24,7 +24,9 @@ describe('mongodb+srv', function () {
         test[1].comment = test[0];
       }
 
-      it(test[1].comment, {
+      // TODO: Remove with NODE-3011
+      const maybeIt = test[1].comment.includes('loadBalanced') ? it.skip : it.only;
```

Comment:
is .only actually what we want here? not just plain `it`?

Reply:
Whoops, that was for me locally. I'll update.

Reply:
Updated.